File: Jack.Model/Model/Calcado.vb
<Serializable()> _
Public Class Calcado

    Public Sub New()

        Codigo = 0
        Numero = 0
        Sexo = String.Empty
        NumeroInicio = 0
        NumeroFim = 0
        MedidaIdade = String.Empty

    End Sub

    Public Overridable Property Codigo As Integer
    Public Overridable Property Numero As Integer
    Public Overridable Property Sexo As String
    Public Overridable Property NumeroInicio As Integer
    Public Overridable Property NumeroFim As Integer
    Public Overridable Property MedidaIdade As String

    Public Overridable ReadOnly Property IdadeInicial As String
        Get
            If MedidaIdade = "M" Then
                Return NumeroInicio.ToString() + " meses"
            Else
                Return NumeroInicio.ToString() + " anos"
            End If
        End Get
    End Property

    Public Overridable ReadOnly Property IdadeFinal As String
        Get
            If MedidaIdade = "M" Then
                Return NumeroFim.ToString() + " meses"
            Else
                Return NumeroFim.ToString() + " anos"
            End If
        End Get
    End Property

    Public Overridable ReadOnly Property SexoDescricao As String
        Get
            If Sexo = "M" Then
                Return "<span style='color:MidnightBlue;'>Masculino</span>"
            Else
                Return "<span style='color:Red;'>Feminino</span>"
            End If
        End Get
    End Property

End Class

File: Jack.Model/Model/Calcado.cs
using System;

namespace Jack.Model
{
    [Serializable()]
    public class Calcado : BaseModel<Calcado>
    {
        private int numero;
        private string sexo;
        private int numeroInicio;
        private int numeroFim; 
        private string medidaIdade;

        public Calcado()
        {
            numero = 0;
            sexo = string.Empty;
            numeroFim = 0;
            numeroInicio = 0;
            medidaIdade = string.Empty;

        }


        public virtual int Numero
        {
            get
            {
                return numero;
            }
            set
            {
                numero = value;
            }
        }

        public virtual string Sexo
        {
            get
            {
                return sexo;
            }
            set
            {
                sexo = value;
            }
        }

        public virtual int NumeroInicio
        {
            get
            {
                return numeroInicio;
            }
            set
            {
                numeroInicio = value;
            }
        }

        public virtual int NumeroFim
        {
            get
            {
                return numeroFim;
            }
            set
            {
                numeroFim = value;
            }
        }

        public virtual string MedidaIdade
        {
            get
            {
                return medidaIdade;
            }
            set
            {
                medidaIdade = value;
            }
        }

        public virtual string IdadeInicial
        {
            get
            {
                if (MedidaIdade == "M")
                {
                    return NumeroInicio.ToString() + " meses";
                }
                else {
                    return NumeroInicio.ToString() + " anos";
                }
            }
        }

        public virtual string IdadeFinal
        {
            get
            {
                if (MedidaIdade == "M")
                {
                    return NumeroFim.ToString() + " meses";
                }
                else {
                    return NumeroFim.ToString() + " anos";
                }
            }
        }

        public virtual string SexoDescricao
        {
            get
            {
                if (Sexo == "M")
                {
                    return "<span style='color:MidnightBlue;'>Masculino</span>";
                }
                else {
                    return "<span style='color:Red;'>Feminino</span>";
                }
            }
        }

    }
}
